ci/options.json

```diff
@@ -20,13 +20,6 @@
     "LanguageVersion": "1.21",
     "PackageRequirement": true
   },
-  {
```

Comment:
let's not remove the entire windows tests, let's just turn off specific failing tests on windows with a todo comment to address later